ui/apps/platform/src/Containers/Policies/Modal/PolicyImport.utils.ts

```diff
@@ -1,31 +1,46 @@
+import { Policy } from 'types/policy.proto';
+import { ImportPolicyResponse } from 'services/PoliciesService';
+
 export const MIN_POLICY_NAME_LENGTH = 5;
 
-export const POLICY_DUPE_ACTIONS = {
-    KEEP_BOTH: 'keepBoth',
-    RENAME: 'rename',
-    OVERWRITE: 'overwrite',
-};
+type PolicyImportError = {
+    type: string;
+    duplicateName: string;
+    validationError?: string;
```

Comment:
From backend response proto, it has either `duplicateName`, or `validationError`, or maybe neither?

https://github.com/stackrox/stackrox/blob/master/proto/api/v1/policy_service.proto#L97-L104

A type union might help with this backend type and the frontend type below. Here is a rough draft:

```ts
type PolicyImportError = PolicyImportErrorDuplicateName | /* and so on */;

type PolicyImportErrorDuplicateName = {
    type: 'duplicate_name';
    duplicateName: string;
} & PolicyImportErrorBase;

// type PolicyImportErrorDuplicateId?

type PolicyImportErrorInvalidPolicy = {
    type: 'invalid_policy';
    validationError: string;
} & PolicyImportErrorBase;

type PolicyImportErrorBase = {
    type: string;
    message: string;
};
```